brainzutils/musicbrainz_db/place.py

```diff
@@ -0,0 +1,85 @@
+from collections import defaultdict
+from mbdata import models
+from sqlalchemy.orm import joinedload
+from brainzutils.musicbrainz_db import mb_session
+from brainzutils.musicbrainz_db.includes import check_includes
+from brainzutils.musicbrainz_db.serialize import serialize_places
+from brainzutils.musicbrainz_db.helpers import get_relationship_info
+from brainzutils.relationships import place as place_rel
+from brainzutils.musicbrainz_db.utils import get_entities_by_gids
+
+
+def get_place_by_id(mbid):
+    """Get place with the MusicBrainz ID.
+
+    Args:
+        mbid (uuid): MBID(gid) of the place.
+    Returns:
+        Dictionary containing the place information.
+    """
+    place = _get_place_by_id(mbid)
+    return place_rel.process(place)
+
+
+def _get_place_by_id(mbid):
+    return fetch_multiple_places(
+        [mbid],
+        includes=['artist-rels', 'place-rels', 'release-group-rels', 'url-rels'],
```

Comment:
Same for other entities (specifying the includes should be done by the user and the library should handle all kinds of includes).